src/java.base/share/classes/java/util/random/package-info.java

```diff
@@ -81,8 +81,8 @@
  * <blockquote>{@code import java.util.random.*;}</blockquote>
  *
  * Then one can choose a specific implementation by giving the name of a generator
- * algorithm to the static method {@link RandomGenerator#of}, in which case the
- * no-arguments constructor for that implementation is used:
+ * algorithm to the static method {@link RandomGenerator#of}, in which case no
+ * seed is specified by the caller:
```

Comment:
Perhaps reword this to:


> ... in which case a {@code RandomGenerator} is constructed without any seed value:

Reply:
An already existing issue in the specification in this file is that, a few lines below, we note that:

> <p> There are three groups of random number generator algorithm provided in Java: the Legacy group, the LXM group, and the Xoroshiro/Xoshiro group.

The "three" groups is misleading I think, since both later in the table as well as the `group()` method implementation on `RandomGeneratorFactory`, we return four distinct values "Legacy", "LXM", "Xoroshiro" and "Xoshiro". Should we reword this part of the documentation to remove the mention of "three"?